app/routes/routes.py

```diff
@@ -34,8 +34,8 @@
 
 
 this_date = datetime.now().strftime("%B, %Y")
-this_month = date.today().month
-this_year = date.today().year
+this_month = date.today().month  # TODO: make this changeable in nav
```

Comment:
tbh i think it's better to move it to __init__.py

Reply:
__init__.py is called once when a package is imported, we want to have those variables available constatly when app is running. If You move it to __init__.py, You will lose it as soon as this script finishes. Correct me if I am wrong.

Reply:
True, if those were constant then it would work, but those are not constant. My bad, you can merge.